Dockerfiles/midstream/run_tests.py

```diff
@@ -6,42 +6,47 @@
 import subprocess
 from os import path
 
+# global variables set 
+OPERATOR_DIR="/tmp/operator_dir/"
+OPERATOR_WORK_DIR="/tmp/test_operator_work_dir/"
+WORK_DIR="/tmp/output/"
+PLAYBOOKS_DIR="/tmp/operator-test-playbooks/"
```

Comment:
playbooks should be `read-only` and not in tmp dir cause people could remove them by mistake.

/tmp is just for temporary files and playbooks and roles are not temporary files.

Reply:
No longer using the tmp.